src/App.tsx

```diff
@@ -1,10 +1,52 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import { useEffect, useState, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Footer } from './components/Footer';
+import { TodoItem } from './components/TodoItem';
+import { Error } from './components/Error';
 
-const USER_ID = 0;
+const USER_ID = 10305;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState('all');
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
why `isLoading` always is true?
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/91411692/63230cb4-9654-4ffa-a4bb-1b00a339e8df)